Search/BinarySearch.js

```diff
@@ -48,8 +48,23 @@ function binarySearchIterative (arr, x, low = 0, high = arr.length - 1) {
   // if low > high => we have searched the whole array without finding the item
   return -1
 }
+function binarySearchIterativeAlternative (arr, x) {
+  let low = -1
```

Comment:
Consider adding a comment stating that these bounds are both exclusive (which is probably less intuitive than the other impl.).